alphadia/workflow/optimization.py

```diff
@@ -88,6 +93,7 @@ def __init__(
         self.workflow.optimization_manager.fit({self.parameter_name: initial_parameter})
         self.has_converged = False
         self.num_prev_optimizations = 0
+        self.num_consecutive_skips = 0
```

Comment:
could this (and others?) be private?

Reply:
I do refer to them outside the class, but only for logging purposes. Should I just move the logging inside the class?

Reply:
I have done this.